src/main.cpp

```diff
@@ -72,6 +72,12 @@ bool cRoot::m_RunAsService = false;
 
 
 #ifndef _DEBUG
+// Because SQLiteCpp uses NULL instead of nullptr, we need to disable the Clang warning for std::signal here
```

Comment:
I assume there's no reason for SQLiteCpp to be here.

Reply:
yep, just a copy-paste failure 👍